api/handler/auth/forget_password_test.go

```diff
@@ -14,10 +14,15 @@ type forget_password_req struct {
 	email string
 }
 
+type forgetPwdMockData struct {
```

Comment:
no need to  create a new request struct for testing real HTTP handler. Once you use the real handler (`ForgetPassword(mockUserStore, mockTokenStore).ServeHTTP(w, r)`), it will also use the real request struct (`ForgetPasswordRequest`)